source/scripts/Utility.js

```diff
@@ -6,3 +6,20 @@ module.exports.queryURL = function(name) {
     var results = new RegExp("[\\?&]" + name + "=([^&#]*)").exec(location.search)
     return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "))
 }
+
+
+module.exports.Point = class Point {
+    constructor(point = {}) {
+        this.x = point.x || 0
+        this.y = point.y || 0
+    }
+    toString() {
+        return this.x + "x" + this.y
+    }
+    toPoint(that = {}) {
```

Comment:
So to actually translate, you'd:

```js
this.position = this.position.translate({x: 1})
```

Which I think is fine. Regardless, I don't think I'll ever use it that way, since I'd just:

```js
this.position.x += 1
```